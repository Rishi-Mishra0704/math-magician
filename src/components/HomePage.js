import React from 'react';
import styles from './css/Homepage.module.css';

const HomePage = () => (
  <div className={styles['homepage-container']}>
    <h3 className={styles['homepage-title']}>Welcome to our page</h3>
    <p className={styles['homepage-text']}>
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Recusandae
      modi ratione in? Nobis explicabo cupiditate doloremque pariatur maxime
      ad, facere accusantium quam fuga tempore! Sequi, autem minima
      perferendis nulla labore expedita accusamus molestias? Nemo, tempora
      maxime perspiciatis sunt illum odio provident, ducimus dolorem iure
      dolores nulla numquam velit repellendus facilis.
    </p>
    <p className={styles['homepage-text']}>
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Recusandae
      modi ratione in? Nobis explicabo cupiditate doloremque pariatur maxime
      ad, facere accusantium quam fuga tempore! Sequi, autem minima
      perferendis nulla labore expedita accusamus molestias? Nemo, tempora
      maxime perspiciatis sunt illum odio provident, ducimus dolorem iure
      dolores nulla numquam velit repellendus facilis.
    </p>
  </div>
);

export default HomePage;
