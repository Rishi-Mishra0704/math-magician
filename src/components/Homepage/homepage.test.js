import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from './HomePage';

it('renders homepage correctly on the DOM', () => {
  const tree = renderer.create(<HomePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
