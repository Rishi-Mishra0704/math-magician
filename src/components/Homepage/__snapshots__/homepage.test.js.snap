// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage correctly on the DOM 1`] = `
<div
  className="homepage-container"
>
  <h3
    className="homepage-title"
  >
    Welcome to our page
  </h3>
  <p
    className="homepage-text"
  >
    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Recusandae modi ratione in? Nobis explicabo cupiditate doloremque pariatur maxime ad, facere accusantium quam fuga tempore! Sequi, autem minima perferendis nulla labore expedita accusamus molestias? Nemo, tempora maxime perspiciatis sunt illum odio provident, ducimus dolorem iure dolores nulla numquam velit repellendus facilis.
  </p>
  <p
    className="homepage-text"
  >
    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Recusandae modi ratione in? Nobis explicabo cupiditate doloremque pariatur maxime ad, facere accusantium quam fuga tempore! Sequi, autem minima perferendis nulla labore expedita accusamus molestias? Nemo, tempora maxime perspiciatis sunt illum odio provident, ducimus dolorem iure dolores nulla numquam velit repellendus facilis.
  </p>
</div>
`;
